core/backend/src/IModelHost.ts

```diff
@@ -572,7 +592,7 @@ export class IModelHost {
    * @internal
    */
   public static get usingExternalTileCache(): boolean {
-    return undefined !== IModelHost.tileCacheService;
+    return undefined !== IModelHost.tileStorage;
```

Comment:
You have deprecated but not removed `tileCacheService`. If an application sets that property then shouldn't `usingExternalTileCache` continue to return `true`?

Reply:
`IModelTileRpcImpl.getTileContent` now uses and depends on `IModelHost.tileStorage` being set, which this check assures

Reply:
We kind of expect people to set either both or only `tileCacheStorage`, as pointed out in the other comment. If someone only sets `tileCacheService`, new frontends won't be able to connect to the tile cache and the backend won't be able to cache any new tiles. I think returning `true` in this case would just mask the problem because people using older frontends wouldn't notice cache misses immediately - it's better if they notice something's wrong sooner rather than later.